packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -263,6 +266,11 @@ class ScrollBehavior {
   /// method returns false, the rebuilds might be optimized away.
   bool shouldNotify(covariant ScrollBehavior oldDelegate) => false;
 
+  /// The platform whose keyboard dismiss behavior physics should be implemented.
```

Comment:
Physics?

Reply:
I've pushed a new version without the "phyiscs" word.